cfg/config.go

```diff
@@ -3,11 +3,98 @@
 // that will be retrieved from  .env or ./cfg/config.yml
 package cfg
 
+import (
+	"bufio"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+// Load configs from a env file & sets them in environment variables .
+func LoadEnvVar() error {
+
```

Comment:
I do not want to look like a code nazi but whitespace right after func sign is redundant.

Reply:
You're not a code nazi, you're a code purity advocate. And you're right)
Corrected.